src/components/Books.js

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import { v4 as uuidv4 } from 'uuid';
+import Addbook from './Addbook';
+import Book from './Book';
+
+const content = [
+  {
+    id: uuidv4(),
+    title: 'THe Flying Bus',
+    author: 'Njamba nene',
+  },
+  {
+    id: uuidv4(),
+    title: 'Life in Crime',
+    author: 'Kiriamiti',
+  },
+  {
+    id: uuidv4(),
+    title: 'River between',
+    author: 'Ngugi wa thiongo',
+  },
+];
+
+const Books = () => (
```

Comment:
[OPTIONAL]

- Instead of naming `Books`, you can name the component `BookList` which will give more idea about the component work and also will not confuse between `Book` and `Books`.

Reply:
Thanks I will consider it.